books-api/Take Book Contract.md

```diff
@@ -0,0 +1,108 @@
+# Take Book Contract
+
+### POST /api/books/{id}/take
+
+Request body:
+```ts
+{
+  сopyId: 1,
+  takenAtUtc: `15.08.2025`
+  sheduledReturnAtUtc: `22.11.2025`
+  // readersIds: [
+  //   {
+  //     corporateEmail: "iivanov1@mail.com",
+  //   },
+  //   {
+  //     corporateEmail: "iivanov2@mail.com",
+  //   },
+  // ]
+}
+```
+
+Response
+```ts
+{
+  id: 1,
```

Comment:
What is this `id`? We shouldn't name properties like this. For instance, when we create a new book we should return `newBookId`. Here I cannot understand what is going to be created, I just take a book, not creating something and naming is not helping as well.